fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -5,11 +5,13 @@ module SharedValues
     end
 
     class TestfairyAction < Action
-      def self.upload_build(ipa, options)
+      def self.upload_build(api_url, ipa, options)
```

Comment:
@joshdholtz would you mind changing the param name to "upload_url" instead of "api_url? We are going to use "api_url" for other purposes very soon. Thanks! 
(I'm Yair from the TestFairy team)

Reply:
Change has been made! Thanks for letting us know about this 🙌